wadas/domain/database.py

```diff
@@ -796,6 +814,133 @@ def sanitize_db(cls):
                 )
                 cls.run_query(stmt)
 
+    @classmethod
+    def get_users(cls):
+        """Method to retrieve users from db"""
+        try:
+            if session := DataBase.create_session():
+                stmt = select(ORMUser.username, ORMUser.email, ORMUser.role)
+                return session.execute(stmt)
+            else:
+                return None
+        except Exception as e:
+            WADASErrorMessage(
+                "Failed to retrieve users data",
+                f"Failed to retrieve user data from db. "
+                f"Please make sure db is healthy and properly configured.\n\nError: {str(e)}",
+            ).exec()
+        finally:
+            session.close()
+
+    @classmethod
+    def get_user_email_and_role(cls, username):
+        """Method to retrieve user email and role from db"""
+
+        try:
+            if session := DataBase.create_session():
+                stmt = select(ORMUser.email, ORMUser.role).where(ORMUser.username == username)
+                result = session.execute(stmt).fetchone()
+
+                if result:
+                    email, role = result
+                    return email, role
+                else:
+                    return None
```

Comment:
Is there any problem just returning the result:
                return session.execute(stmt).fetchone()
?

Otherwise, possible alternatives:
                return session.execute(stmt).fetchone() or None

                return result if result := session.execute(stmt).fetchone() else None